ImageLounge/src/DkGui/DkShortcuts.cpp

```diff
@@ -68,13 +68,16 @@ static bool resolveShortcut(const QWidget *target, const QSet<const QWidget *> &
 
     // breadth-first resolution descending into children
     // qDebug().noquote() << "[shortcut] resolving:" << keySeq << QString(' ').repeated(depth * 2) << target->metaObject()->className() << target->objectName();
-    for (auto *a : target->actions())
-        if (QKeySequence::ExactMatch == a->shortcut().matches(keySeq)) {
-            a->trigger();
-            return true;
-        }
-
-    for (auto *o : target->children()) {
+    for (QAction *a : target->actions())
+        if (a->isEnabled())
```

Comment:
Maybe do this to save one level of indentation.
```suggestion
        if (!a->isEnabled()) {
            continue;
        }
```